odl/operator/__init__.py

```diff
@@ -34,3 +34,5 @@
 __all__ += pspace_ops.__all__
 
 from . import oputils
```

Comment:
That is a rather general statement about these. In general they should all be removable, but we may need to change to relational paths `from . import XXX` instead then. Imo we should make an issue of this. This would also help making the global namespace smaller.